.eslintrc.json

```diff
@@ -1,68 +1,51 @@
 {
-  "plugins": [
-      "react",
-      "prettier"
+  "plugins": ["react", "prettier"],
+  "extends": [
+    "react-app",
+    "react-app/jest",
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:react/jsx-runtime"
+  ],
+  "rules": {
+    "semi": ["error", "always"],
+    "quotes": ["error", "single"],
+    "no-var": "error",
+    "jsx-quotes": "error",
+    "no-multi-spaces": "error",
+    "react/prop-types": "off",
+    "react/no-array-index-key": "error",
+    "react/jsx-pascal-case": "error",
+    "react/jsx-closing-bracket-location": "error",
+    "react/jsx-closing-tag-location": "error",
+    "react/jsx-tag-spacing": "error",
+    "react/jsx-curly-spacing": "error",
+    "react/jsx-boolean-value": "error",
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
```

Comment:
Just for info: The concept of this rule was to only allow files with jsx extension to have jsx syntax, adding, js means both files can have jsx. This is not so important if you prefer to use .js file to have jsx as well. So this rule can be removed which is equivalent to your changes.